project/Http4sPlugin.scala

```diff
@@ -102,6 +246,27 @@ object Http4sPlugin extends AutoPlugin {
       }
     ).flatten
 
+  def latestPerMinorVersion(file: File): Map[(Int, Int), Version] =
+    JGit(file).tags.collect {
+      case ref if ref.getName.startsWith("refs/tags/v") =>
+        Version(ref.getName.substring("refs/tags/v".size))
+    }.foldLeft(Map.empty[(Int, Int), Version]) {
+      case (m, Some(v)) =>
+        def toMinor(v: Version) = (v.major, v.subversions.headOption.getOrElse(0))
+        def patch(v: Version) = v.subversions.drop(1).headOption.getOrElse(0)
+        def milestone(v: Version) = v.qualifier match {
+          case Some(q) if q.startsWith("-M") => q.substring(2).toInt
+          case Some(q) if q.startsWith("-RC") => q.substring(3).toInt + 1000000
```

Comment:
I may be missing it, whats with the `+ 1000000`

Reply:
I wanted to sort RCs after milestones.  I should turn this into a tuple and Do It Right.